components/automate-chef-io/data/docs/api-static/03-tags.swagger.json

```diff
@@ -15,14 +15,22 @@
     {
       "name": "Applications",
       "tags": ["service_groups", "retention"]
-    }
+    },
+    {
+      "name": "Identity and Access Management",
+      "tags": ["policies", "roles", "projects", "tokens"]
```

Comment:
I copied this from https://github.com/chef/automate/pull/2689 but would like them alphabetized when that merges 🤓

Reply:
If it were a multiline list it might help merge conflicts if more things are going to get added.